src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/xlf/MicrosoftCodeQualityAnalyzersResources.ja.xlf

```diff
@@ -104,7 +104,7 @@
       </trans-unit>
       <trans-unit id="DoNotCatchGeneralExceptionTypesMessage">
         <source>Modify '{0}' to catch a more specific allowed exception type, or rethrow the exception</source>
-        <target state="translated">'{0}' を変更してより具体的な許可された例外の種類をさらにキャッチするか、例外を再スローします</target>
+        <target state="translated">{0}' を変更してより具体的な許可された例外の種類をキャッチするか、例外を再スローしてください</target>
```

Comment:
The single quote appears **only** after `{0}`, but not before. Is this intentional?

Reply:
No, this is not intentional. Fixed.

Reply:
https://github.com/dotnet/roslyn-analyzers/pull/4753/commits/4b6814f67e6f510459868de9b2aad57d1a80264f